starter-code/public/stylesheets/footer.css

```diff
@@ -0,0 +1,15 @@
+.footer-container {
+  display: flex;

```

Comment:
cq--
You should use not a display:flex, but fixed position, here:
position: fixed;
bottom: 0;  
right: 0;